src/uu/sort/src/sort.rs

```diff
@@ -1957,47 +1951,37 @@ enum Month {
 }
 
 /// Parse the beginning string into a Month, returning [`Month::Unknown`] on errors.
-fn month_parse(line: &str) -> Month {
-    let line = line.trim();
-
-    const MONTHS: [(&str, Month); 12] = [
-        ("JAN", Month::January),
-        ("FEB", Month::February),
-        ("MAR", Month::March),
-        ("APR", Month::April),
-        ("MAY", Month::May),
-        ("JUN", Month::June),
-        ("JUL", Month::July),
-        ("AUG", Month::August),
-        ("SEP", Month::September),
-        ("OCT", Month::October),
-        ("NOV", Month::November),
-        ("DEC", Month::December),
+fn month_parse(line: &[u8]) -> Month {
```

Comment:
Is this function implemented in this way for performance reasons? My naive (?) approach would be to make `line` uppercase and then match on the first three characters, something like:
```rust
    let line = line.trim_ascii_start().to_ascii_uppercase();

    match line.get(..3) {
        Some(b"JAN") => Month::January,
        // other months
        _ => Month::Unknown,
    }
```

Reply:
You're right ! it should be faster to do so.
I will make the change.

Note that in any case, this is not correct in locale-aware context, so it will change again